### DISCUSSION TITLE
How can I cite the repo?

### DISCUSSION THREAD

**Discussion by jaime-cespedes-sisniega** [bentoml/BentoML#3998] [Q&A] [answered]
I was checking the repository and the documentation but I haven't found a way to cite it.

How can I do it? I was looking for some BibTeX format.

Thanks in advance.

**Comment by FogDong** [comment]
Yeah we can add cite to BentoML as well. cc @aarnphm

**Reply by aarnphm** [reply]
Ye I think we can add it

**Reply by aarnphm** [reply]
https://github.com/bentoml/BentoML/pull/4019

**Comment by aarnphm** [comment, marked as answer]
Hey @jaime-cespedes-sisniega, I have added the citation into the main branch. Thank you for using BentoML in your research.

**Reply by jaime-cespedes-sisniega** [reply]
Thank you!
